APMrover2/GCS_Mavlink.cpp

```diff
@@ -1582,7 +1582,7 @@ void Rover::gcs_send_text_fmt(MAV_SEVERITY severity, const char *fmt, ...)
     char str[MAVLINK_MSG_STATUSTEXT_FIELD_TEXT_LEN] {};
     va_list arg_list;
     va_start(arg_list, fmt);
-    hal.util->vsnprintf((char *)str, sizeof(str), fmt, arg_list);
+    hal.util->vsnprintf(reinterpret_cast<char *>(str), sizeof(str), fmt, arg_list);
```

Comment:
Why reinterpret_cast this?  I used static_cast and it compiled fined.  That doesn't mean its working of course but generally reinterpret_cast should be avoided.